### ISSUE TITLE
Add term "Body Ache" as a Constitutional Symptom

### ISSUE THREAD

**Issue by JasonBarnettNC** [obophenotype/human-phenotype-ontology#5833]
**Preferred term label:** Body Ache

**Synonyms** Body Ache or Pain

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)** Constitutional Symptom https://hpo.jax.org/app/browse/term/HP:0025142

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

@pnrobinson  - I'm adding this request from our discussion yesterday, open to better suggestions for placement - purpose is to capture "body ache" as a non-localized symptom. Perhaps this could also be a synonym for "Pain: HP:0012531" Thank you

**Comment by pnrobinson** [comment]
Definition: Body ache is a complaint that is often used to denote vague symptoms of mild fatigue, lethargy, or dull aches. We will define it here to mean a dull and poorly localizable pain that is described by the affected individual to affect multiple joints or body parts or even the entire body.

**Comment by pnrobinson** [comment]
added as HP:0033047, this will be online in August. @JasonBarnettNC
